docs/release-notes/6.8.md

```diff
@@ -0,0 +1,39 @@
+# Release notes for Gluster 6.8
+
+This is a bugfix release. The release notes for [6.0](6.0.md), [6.1](6.1.md),
+ [6.2](6.2.md), [6.3](6.3.md), [6.4](6.4.md), [6.5](6.5.md) [6.7](6.7.md) and [6.8](6.8.md)
```

Comment:
@harigowtham You forgot a comma(,) after `[6.5](6.5.md)`. So it'll be `[6.5](6.5.md), [6.7](6.7.md)` and not `[6.5](6.5.md) [6.7](6.7.md)`.

Reply:
Thanks for the catch, will update.

Reply:
done, I'll resolve this conversation. If you have any other comments, please feel free to open another.